packages/fast-components-react-base/.npmignore

```diff
@@ -4,7 +4,12 @@ __test__/
 *.test.*
 
 # Source files
+app/
+coverage/
 src/
+www/
 
 # babel config
-.babelrc
\ No newline at end of file
+.babelrc
+tsconfig.json

```

Comment:
we might also want to add `webpack.config.js` to packages that use webpack

Reply:
good idea, updated.